qiskit/circuit/__init__.py

```diff
@@ -54,81 +78,825 @@
    circ.draw('mpl')
 
 
-Supplementary Information
-=========================
+.. _circuit-definitions:
+
+Circuit concepts and definitions
+================================
+
+There is a lot of specialized terminology around quantum circuits.  Much of this is common in
+quantum-computing literature, while some is more specific to quantum software packages, and a small
+amount specific to Qiskit.  This is an alphabetical listing of some of the important concepts as a
```

Comment:
```suggestion
quantum computing literature, while some is more specific to quantum software packages such as Qiskit.  This is an alphabetical listing of some of the important concepts as a
```

Reply:
I can make this clearer if you like, but your suggestion changes the intended meaning. The idea is that some of the terminology is totally general, some is jargon related to all quantum software, some is jargon related specifically to Qiskit - that's why the last two elements were separated.

(It's also neither here nor there, but the removed hyphen wasn't wrong - "quantum-computing" here is a compound adjective modifying the noun "literature", and compound adjectives are typically hyphenated.)

Reply:
Sounds good to me.